.gitignore

```diff
@@ -35,6 +35,9 @@ cmake-build-release/
 # IntelliJ
 out/
 
+#vscode
+.vscode
```

Comment:
Is this a file only?

Reply:
It definitely should be a catalog

Reply:
Actually it might stay as it is. If you don't append a slash, the pattern will match both files and the contents of directories with that name.

Reply:
ok checks out.
`.vscode` will indeed ignore a file of that name, and a directory structure starting with `.vscode/`